### PR TITLE
Adjust zli development instructions in README [CWC-2487]

### PR DESCRIPTION
**Pull request by tflpd**
## Description of the change

Add more elaborate README instructions for incoming engineers

## Testing

Describe how to test this PR....

**backend branch:** 
**bzero branch:** 
**charts branch:** 

#### Ready to run system tests?

- [x] Yes

## Relevant release note information

Release Notes: Adjust zli development instructions in README

## Related JIRA tickets

Relates to JIRA: CWC-2487

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [x] No

If yes, please explain:

Does this PR add a new/modifies the name/arguments of a `zli` command?

- [ ] Yes
- [x] No

If yes, please provide the docs/zli-reference-manual branch reflecting the changes:

### CODE REVIEW

**Review by johncmerfeld — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,12 +60,39 @@ brew install bastionzero/tap/zli
...
+```
+
+Finally, you will need to install the required packages
+```
+npm install
+```
+
 ### Build
 
+Now that everything is ready you can execute any `zli` command you wish
```
Comment (chad-cwc) [inline README.md:82]: Since this is the build section, I suggest it has something about building the code.

Comment (tflpd) [reply README.md:82]: fair, but this essentially builds as well.. maybe adjust it to build and run?

Comment (chad-cwc) [reply README.md:82]: True. Maybe "build and run" and maybe add something about "See other build options in package.json" although people familiar with node applications should be aware of where to look. It's best not to assume, though, I think.

Comment (tflpd) [reply README.md]: 8bb3700c

**Review by chad-cwc — APPROVED** [review]

**Pull request merged**